### DISCUSSION TITLE
New version not on docker hub yet?

### DISCUSSION THREAD

**Discussion by marcelweikum** [firefly-iii/firefly-iii#9075] [Firefly III] [answered]
Hey guys,
I am using docker compose with the docker-compose.yml and I tried to install the new 6.1.19, because my firefly installation tells me to update. I can find the 6.1.19 on packagist but I can't find it on dockerhub, the newest version is "develop".

Is the 6.1.19 not officially released yet or is it just missing on docker hub?

**Comment by JC5** [comment, marked as answer]
It's not there yet, I forgot to start the job 😅. Should be there later today.

**Reply by marcelweikum** [reply]
Oh, thank you :+1:
